src/ConnectedMode/Binding/CSharpVBBindingConfig.cs

```diff
@@ -49,10 +47,18 @@ internal CSharpVBBindingConfig(FilePathAndContent<string> globalConfig, FilePath
 
         public void Save()
         {
+            EnsureParentDirectoryExists(GlobalConfig.Path);
             fileSystem.File.WriteAllText(GlobalConfig.Path, GlobalConfig.Content);
 
             var serializedAdditionalFile = Serializer.ToString(AdditionalFile.Content);
+            EnsureParentDirectoryExists(AdditionalFile.Path);
             fileSystem.File.WriteAllText(AdditionalFile.Path, serializedAdditionalFile);
         }
+
+        private void EnsureParentDirectoryExists(string filePath)
```

Comment:
This method is duplicated in the two binding config implementations. However, it's so small I didn't think it was worth factoring out elsewhere.